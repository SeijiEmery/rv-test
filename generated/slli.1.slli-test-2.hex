0000000    93  90  80  00  13  11  81  00  93  91  81  00  13  00  00  00
0000020    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    73  00  10  00                                                
0000044
