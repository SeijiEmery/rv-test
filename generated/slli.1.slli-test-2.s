slli x1, x1, 8
	slli x2, x2, 8
	slli x3, x3, 8
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
