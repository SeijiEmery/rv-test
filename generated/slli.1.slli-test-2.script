load /x 0 generated/slli.1.slli-test-2.hex
writereg 1 7
writereg 2 1
writereg 3 1024
setpc 0
run 100000
readreg 1
readreg 2
readreg 3