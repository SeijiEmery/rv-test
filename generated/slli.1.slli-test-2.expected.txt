tests/slli.test.s:11
R1 = 1792
R2 = 256
R3 = 262144
